module/actor/actor-sheet.js

```diff
@@ -378,9 +381,22 @@ ${parseInt(data.data.movement.shift.value)} ${game.i18n.localize("DND4EBETA.Move
 		}
 		else if(this.object.data.data.powerGroupTypes === "type") {
 			return {
-				class: { label: "Class Power", items: [], dataset: {type: "class"} },
-				race: { label: "Racial Power", items: [], dataset: {type: "race"} },
+				class: { label: "DND4EBETA.Class", items: [], dataset: {type: "class"} },
+				race: { label: "DND4EBETA.Race", items: [], dataset: {type: "race"} },
+				paragon: { label: "DND4EBETA.Paragon", items: [], dataset: {type: "paragon"} },
+				epic: { label: "DND4EBETA.Epic", items: [], dataset: {type: "epic"} },
+				theme: { label: "DND4EBETA.Theme", items: [], dataset: {type: "theme"} },
+				feat: { label: "DND4EBETA.Feat", items: [], dataset: {type: "feat"} },
+				item: { label: "DND4EBETA.PowerItem", items: [], dataset: {type: "item"} },
+				//item: { label: "DND4EBETA.PowerUtil", items: [], dataset: {type: "utility"} },
```

Comment:
Utility grouping should stay in for legacy, since you are keeping util i the options for legacy powers, otherwise they will all get grouped under "other"

Reply:
I kind of liked the "other" result, as it doesn't stop the powers using the legacy setting from displaying, but it does indicate that they are in need of an update. You're confident it's undesirable?

Reply:
I could go either way.  We can have a try your way and see if people complain ;)